libnetwork/netavark/config.go

```diff
@@ -155,13 +155,13 @@ func (n *netavarkNetwork) networkCreate(newNetwork *types.Network, defaultNet bo
 
 	switch newNetwork.Driver {
 	case types.BridgeNetworkDriver:
-		err = internalutil.CreateBridge(n, newNetwork, usedNetworks, n.defaultsubnetPools)
-		if err != nil {
-			return nil, err
-		}
-		// validate the given options, we do not need them but just check to make sure they are valid
+		// validate the given options
 		for key, value := range newNetwork.Options {
 			switch key {
+			case "com.docker.network.driver.mtu":
+				newNetwork.Options[types.MTUOption] = value
+				fallthrough
+
```

Comment:
you should to remove the docker option from the map otherwise we end up with two times the same option in the config which is undesirable.

Reply:
these docker options are applicable only for bridge drivers right?

Reply:
Addressed review comments. PTAL

Reply:
I've added only to the bridge driver option as per this doc
https://docs.docker.com/engine/reference/commandline/network_create/#bridge-driver-options